lib/pthreads/pthreads.nit

```diff
@@ -19,6 +19,7 @@ module pthreads is
 	cflags "-pthread"
 	ldflags "-pthread"
 	pkgconfig "bdw-gc"
+	new_annotation threaded
```

Comment:
I the next PR, you should document the annotation in this module or the group's README.